cmd/createredis.go

```diff
@@ -31,12 +51,22 @@ Cobra is a CLI library for Go that empowers applications.
 This application is a tool to generate the needed files
 to quickly create a Cobra application.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("create called")
+		var tr tcloudredis.TcloudRedis
+		err := redisParseConfig(&tr)
+
+		if err == nil {
```

Comment:
it's go convention to handle the error first, e.g.
```suggestion
err := redisParseConfig(&tr)
if err != nil {
  return err
}
```